weblab/entities/models.py

```diff
@@ -16,6 +16,11 @@ class Entity(models.Model):
         (VISIBILITY_RESTRICTED, 'Restricted'),
         (VISIBILITY_PUBLIC, 'Public')
     )
+    VISIBILITY_HELP = (
+        'Public = anyone can view\n'
+        'Restricted = logged in users can view\n'
```

Comment:
Should this not be public/private/group or something? Not sure why logged in users are a privileged group (anyone can make an account right?)

Reply:
Good point. We used to have restricted when it was harder to make an account; with #40 it probably becomes redundant. I'll make a note there.